src/Helper/Asset.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zendframework for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\View\Helper;
+
+use Zend\View\Exception;
+
+/**
+ * View helper plugin to fetch asset from resource map.
+ */
+class Asset extends AbstractHelper
```

Comment:
Please add the helper in the DocBlock of [`PhpRenderer`](https://github.com/zendframework/zend-view/blob/master/src/Renderer/PhpRenderer.php#L34) for code completion in IDEs.